File: OPENAPI-REST-API/swagger-client/dart/lib/model/mail_schema_extra_info_tables.dart
part of swagger.api;

class MailSchemaExtraInfoTables {
  
  MailExtraInfoTable mail = null;

  MailTutorialsTable tutorials = null;

  MailSchemaExtraInfoTables();

  @override
  String toString() {
    return 'MailSchemaExtraInfoTables[mail=$mail, tutorials=$tutorials, ]';
  }

  MailSchemaExtraInfoTables.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    mail = new MailExtraInfoTable.fromJson(json['mail']);
    tutorials = new MailTutorialsTable.fromJson(json['tutorials']);
  }

  Map<String, dynamic> toJson() {
    return {
      'mail': mail,
      'tutorials': tutorials
     };
  }

  static List<MailSchemaExtraInfoTables> listFromJson(List<dynamic> json) {
    return json == null ? new List<MailSchemaExtraInfoTables>() : json.map((value) => new MailSchemaExtraInfoTables.fromJson(value)).toList();
  }

  static Map<String, MailSchemaExtraInfoTables> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, MailSchemaExtraInfoTables>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new MailSchemaExtraInfoTables.fromJson(value));
    }
    return map;
  }
}

File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/MailSchemaExtraInfoTables.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// Additional information tables for the mail service.
  /// </summary>
  [DataContract]
  public class MailSchemaExtraInfoTables {
    /// <summary>
    /// Gets or Sets Mail
    /// </summary>
    [DataMember(Name="mail", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "mail")]
    public MailExtraInfoTable Mail { get; set; }

    /// <summary>
    /// Gets or Sets Tutorials
    /// </summary>
    [DataMember(Name="tutorials", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "tutorials")]
    public MailTutorialsTable Tutorials { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class MailSchemaExtraInfoTables {\n");
      sb.Append("  Mail: ").Append(Mail).Append("\n");
      sb.Append("  Tutorials: ").Append(Tutorials).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
